web-console/knikubevirt/Create-vm/wizard-import/wizard-import.md

```diff
@@ -0,0 +1,73 @@
+# Create A Virtual Machine
+
+## Import VM
+
+The ‘Import virtual machine’ action will be available in the ‘Create virtual machine’ dropdown.
+
+![Empty state](img/empty-state.png)
```

Comment:
Just thinking about the terminology: I wonder if we should use the word "wizard" (even though that's what it is technically) but rather we could sort of treat the wizard as the default option, and only specify when you would prefer to use YAML instead... for example: "Create, Create from YAML, Import"

Reply:
I like that, seems like a nice way to shorten things and be a bit more clear.

Reply:
Hi @rileyhuston, thanks for the comment, but I kind of think that the word ‘wizard’ gives the user a sense of something familiar towards what they are heading to, as opposed to just 'create' or even 'create new'. Maybe we can advise with @lizsurette  and @andybraren about this. Thanks.

Reply:
I've gone back and forth on this a number of times. Ultimately I'm fine with either :) @beanh66 any strong opinions on using "Create" or "Create Wizard" when launching the wizard version of creating a VM?

Reply:
@bmignano and I discussed this for a while too. The long-term vision for the Console is for there to be a default form/wizard-based path to create every object, meaning a single blue button to click without a dropdown that mentions a "wizard" at all, similar to what Riley suggested.

In that beautiful future world, users would then have the option to easily switch between the wizard & YAML "views" in-page rather than choosing YAML up-front. Import would (probably) then be something chosen in Step 0 of the wizard (similar to Liz's design from long, long ago).

But that's long-term and debatable. Short-term we agree that either approach is clear enough to be fine.

To align just a little closer to the only other Create dropdown in today's Console (on the Secrets page) we'd suggest the below, but it's still not perfect. We might have too many cooks here - I'm cool with anything. 🙂

```
Create:
- Virtual machine
- Import virtual machine
- From YAML
```

![image](https://user-images.githubusercontent.com/9122899/62650176-46d77f80-b924-11e9-811e-427bf0c2820f.png)

Reply:
I'm not of a strong opinion here either, consistency is really the only thing I'd prioritize.
I will point out that we broke out the import from the Source dropdown b/c it wasn't as clear to a user who wants to import that they need to go through the Create flow to do it. Maybe that can be further tested though.

Reply:
Thanks!